caf-audit-ui/README.md

```diff
@@ -0,0 +1,11 @@
+# Audit Web Service UI
+
+This Swagger UI project builds a documentation web application for the [CAF Audit Web Service](https://github.com/CAFAudit/audit-service/tree/develop/caf-audit-service). It allows users to see what operations the web service exposes as well as documentation relating to those operations and the web service in general.
+
+## Usage
+
+To get started, deploy the [web service docker container](https://github.com/CAFAudit/audit-service/tree/develop/caf-audit-management-service-container). The web service UI can then be accessed at the following URL:
```

Comment:
Obviously we don't have an Audit Web Service Container yet, however this URL is definitely wrong as we no longer have an "caf-audit-management-service-container".

Once this URL is update, this is probably OK once the Web Service becomes available.